integration-test/src/config/.env.prod

```diff
@@ -1,2 +1,3 @@
 AFM_HOST=https://api.platform.pagopa.it/afm/calculator-service/v1
+PSP_WHITELIST=PPAYITR1XXX,BPPIITRRXXX,CIPBITMM,UNCRITMM,BNLIITRR,BCITITMM,BIC36019
```

Comment:
Do we use this file?

Reply:
probably not but since it was already there I aligned it